projects/docs-tools/src/modules/type-definitions/property-definitions.component.html

```diff
@@ -70,9 +70,20 @@
 <ng-template let-property #descriptionTemplate>
   <div
     *ngIf="!property.isOutput && !property.isOptional && showOptionalStatus"
-    class="sky-docs-property-definition-label sky-text-danger"
+    class="sky-docs-property-definition-label"
   >
-    Required.
+    <sky-status-indicator indicatorType="danger" descriptionType="danger"
+      >Required.</sky-status-indicator
+    >
+  </div>
+  <div
+    *ngIf="property.isPreview"
+    class="sky-docs-property-definition-preview-warning sky-docs-property-definition-label"
+  >
+    <sky-status-indicator indicatorType="warning" descriptionType="warning"
```

Comment:
This status indicator (and message) are used in a few places. Can we create a new component?

Reply:
Done